docs/src/examples/components/Portal/Types/PortalExampleControlled.shorthand.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react'
+import { Button, Divider, Header, Label, Portal } from '@stardust-ui/react'
+
+class PortalExampleControlled extends React.Component<any, any> {
+  public state = {
+    log: [],
+    logCount: 0,
+    open: false,
+  }
+
+  public render() {
+    const { log, logCount, open } = this.state
+    const content = open ? 'Close Portal' : 'Open Portal'
+
+    const controls = (
```

Comment:
`controls` and `portalContent` can also be inlined into the render method to save some lines.

Reply:
I think my approach makes it more clear how to actually use the components since the `controls` and `portalContent` are lengthy and users can get confused; that's why I did it like this... Anyway, I made the change as suggested